### DISCUSSION TITLE
Resuming training with new training dataset

### DISCUSSION THREAD

**Discussion by rathishkumar** [facebookresearch/detectron2#3139] [Q&A]
I have trained model and deployed, based on the model performance, I need to add additional data (which are not part of original training set). Can I use the `resume=True` option to add this new data or do I need to add this data to original training set and start from scratch?

This new data is similar to old data.

**Comment by GitHub-Crow** [comment]
Both of them are not the best way , you can registry a custom dataset, and add to `config.yaml`, maybe this is a better way to add new data.

**Comment by ihholmes-p** [comment]
Hi Rathish, did you get it working? I haven't found any good answers in the discussions. I guess Detectron2 wasn't meant to be trained like this? There's also no explanation about this in the documentation.
